v2/samples/direct_method_invocation_sample/README.md

```diff
@@ -0,0 +1,81 @@
+Azure IoT Edge V2 - Direct Method Invocation Sample (Linux only for now) 
+=====================================================================
+
+Overview
+--------
+
+This sample showcases how one can invoke a direct method from one module to another module 
+or a downstream device on [Azure IoT Edge V2](https://docs.microsoft.com/en-us/azure/iot-edge/how-iot-edge-works). 
+
+It uses the following simple scenario -
+There is an edge deployment consisting of 2 custom modules -
+1. An engine simulator module produces data which includes the engine RPM. It also exposes 
+a Reset method, which if called, resets the engine to its initial value.
+2. An analyzer module analyzes the data sent by the engine simulator module and invokes a 
+Reset direct method call on the engine simulator module if the RPM value is too high.
+
+Invoking direct methods
+--------------
+
+A module can invoke a direct method on another module or a downstream device (i.e. a device
+connected to the same EdgeHub), by using its EdgeHubConnectionString. It cannot invoke direct methods
+on modules on other edge devices, or devices not connected to the same EdgeHub (unless it uses an 
+IotHub scoped connection string).
+
+Here are the steps to invoke a direct method from one module to another -
+
+- You need to use the ServiceClient to invoke direct methods from a module. The ServiceClient is in the Microsoft.Azure.Devices nuget package which is not added by default, so you need to manually add that. Make sure to add a reference to a preview* version of the nuget package, to get modules support. 
```

Comment:
Can you include a CLI statement that shows exactly what you are asking them to do here?

Reply:
Sure..